reps/types.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Callable, Dict
+
+
+Items = Dict[str, Any]
```

Comment:
Is there a better name you could give this type? (I realize it's echoing the variable name at the moment, but it feels like a tweak to both might improve readability.)

Reply:
Sounds reasonable to me!